### ISSUE TITLE
no ebuild to satisfy dnspyhton

### ISSUE THREAD

**Issue by latot** [pentoo/pentoo-overlay#46]
Hi again, i was doing an update and i get this message, here the log:

```
emerge: there are no ebuilds to satisfy "virtual/dnspython[python_targets_python2_7(-)?,python_targets_python3_4(-)?,-python_single_target_python2_7(-),-python_single_target_python3_4(-)]".
(dependency required by "net-analyzer/dnsrecon-0.8.9-r2::pentoo" [installed])
(dependency required by "pentoo/pentoo-analyzer-2015.3::pentoo" [installed])
(dependency required by "pentoo/pentoo-2015.1::pentoo[analyzer]" [installed])
(dependency required by "@selected" [set])
(dependency required by "@world" [argument])
```

Cya.

**Comment by blshkv** [comment]
when did you sync last time? I was fixed couple weeks ago

**Comment by latot** [comment]
yes u.u

```
 * Syncing selected overlay(s)...
 * Running Git... # ( cd /var/lib/layman/pentoo  && /usr/bin/git pull )
Already up-to-date.
 * 
 * Succeeded:
 * ------
 * Successfully synchronized overlay "pentoo".
 * 
```

**Comment by blshkv** [comment]
run emerge --sync as well ;-)

**Comment by latot** [comment]
i do it too D:

**Comment by blshkv** [comment]
oh, Gentoo renamed it to virtual/python-dnspython...

**Comment by blshkv** [comment]
should be fixed by 1a0b6bb4787ba378f9133bbef670c94f652dd915

**Comment by latot** [comment]
yay! fixed thx.
